packages/odyssey-ie-11-sandbox/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@okta/odyssey-ie-11-sandbox",
+  "version": "0.9.1",
+  "description": "Testing for IE11",
+  "author": "Okta, Inc.",
+  "homepage": "https://github.com/okta/odyssey#readme",
+  "license": "Apache-2.0",
+  "main": "dist/index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/okta/odyssey",
+    "directory": "packages/odyssey-ie-11-sandbox"
+  },
+  "scripts": {
+    "start": "webpack serve  --hot --open --mode development",
+    "build": "webpack --mode production"
```

Comment:
How do you envision these two builds being used as part of our workflow? Are there any differences in the `--mode development` flag that will differ from the production polyfill and bundling for IE11?

Reply:
the "start" build is for normal development. The production packed "build" is currently unused, but could be built and deployed manually if necessary.